app/models/user.rb

```diff
@@ -18,4 +18,25 @@ def update_without_password(params, *options)
     clean_up_passwords
     result
   end
+
+  def merge_cart(session_cart)
```

Comment:
```suggestion
  def merge_cart!(session_cart)
```
true/false返さないタイプなら!つけたい